src/server/permissions/AppPermissions.ts

```diff
@@ -109,6 +109,12 @@ export const AppPermissions = {
     },
 };
 
+/**
+ * @description
+ * Default permissions for apps
+ * Used to ensure backward compatibility with apps
+ * that were developed before the permission system was introduced.
+ */
```

Comment:
I could swear there was a description for this here :eyes: Well, my memory failed me. Thanks for adding

Reply:
Haha, it happens a lot to me as well. 🙈